AI_Graphics/AI_Graphics/AIGraphics.cs

```diff
@@ -116,10 +116,16 @@ private IEnumerator Start()
             
             _inspector = new Inspector.Inspector(this);
 
-            // It takes some time to fully loaded in studio to save/load stuffs.
-            yield return new WaitUntil(() => {
-                return (KoikatuAPI.GetCurrentGameMode() == GameMode.Studio) ? KKAPI.Studio.StudioAPI.InsideStudio && _skyboxManager != null : true;
-            });
+            if (KoikatuAPI.GetCurrentGameMode() == GameMode.Maker) {
+                yield return new WaitUntil(() => {
+                    return (KoikatuAPI.GetCurrentGameMode() == GameMode.Maker && _skyboxGO == null);
```

Comment:
isn't the maker check redundant? we wouldn't have been here otherwise.  don't quite understand what this fixes. is this a fix for entering maker a second time and aigraphics breaking? what actually breaks?

Reply:
old 119-122: This used to be there to load default preset files. Since they're gone I commented it.
new 119-121: This condition will only run in Main Character Creation Menu. Addtional Information is in next comment.